proto/substrait/algebra.proto

```diff
@@ -619,7 +620,7 @@ message WriteRel {
   // The type of operation to perform
   WriteOp op = 4;
 
-  // The relation that determines the records to add/remove/modify
```

Comment:
Modification using this particular method is still reasonable -- one would use this if you had ids to merge against.

Reply:
In current state it is not possible. Right?
We can modify the comment if we modify this proto.

Reply:
ok got it, will revert the changes related to WriteRel.